main/py-dbus/APKBUILD

```diff
@@ -1,39 +1,68 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=py-dbus
-pkgver=1.2.0
-pkgrel=2
+pkgver=1.2.4
+pkgrel=0
 pkgdesc="Python bindings for DBUS"
 url="http://www.freedesktop.org/wiki/Software/DBusBindings"
 arch="all"
 license="GPL LGPL"
-depends="python2"
-depends_dev="py-dbus"
-makedepends="dbus-glib-dev python2-dev"
-subpackages="$pkgname-dev $pkgname-doc"
+depends=""
+#depends_dev="py-dbus"
+makedepends="dbus-glib-dev python2-dev python3-dev automake autoconf-archive"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3 $pkgname-doc $pkgname-dev"
 source="http://dbus.freedesktop.org/releases/dbus-python/dbus-python-$pkgver.tar.gz"
 
 _builddir="$srcdir"/dbus-python-$pkgver
 
 prepare() {
-	cd "$_builddir"
-	update_config_sub || return 1
+	local python; for python in python2 python3; do
+		cp -r "$_builddir" "$_builddir-$python"
+	done
 }
-build() {
-	cd "$_builddir"
-	./configure \
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py2() {
+	replaces="$pkgname"
+	_py python2
+}
+
+_py3() {
+	_py python3
+}
+
+_py() {
+	local python="$1"
+	pkgdesc="$pkgdesc (for $python)"
+	depends="$depends $python"
+	# should this line be here ?
+	# install_if="$pkgname-$pkgver-r$pkgrel $python"
+	cd "$_builddir-$python"
+	PYTHON=$python ./configure \
```

Comment:
i think we should build the thing from `build()`, not during subpackage split function.